<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rocketeClass</class>
 <widget class="QMainWindow" name="rocketeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rockete - libRocket RML/RCSS file editor</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTabWidget" name="codeTabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>0</height>
            </size>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Triangular</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="tabsClosable">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>New...</string>
            </attribute>
           </widget>
          </widget>
          <widget class="QTabWidget" name="bottomTabWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>128</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Triangular</enum>
           </property>
           <property name="currentIndex">
            <number>3</number>
           </property>
           <property name="tabsClosable">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>Console</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Attributes</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QTreeView" name="attributeTreeView"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Properties</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QTreeView" name="propertyTreeView"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="currentToolTab">
            <attribute name="title">
             <string>Current tool</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="RenderingView" name="renderingView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionImport_fonts"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuBackground">
     <property name="title">
      <string>Background</string>
     </property>
     <addaction name="actionSet_from_file"/>
     <addaction name="separator"/>
     <addaction name="actionStreched"/>
    </widget>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionZoom_reset"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_grid"/>
    <addaction name="actionGrid_scale"/>
    <addaction name="menuBackground"/>
    <addaction name="separator"/>
    <addaction name="actionSet_screen_size"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionUnselect"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="rockete.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="rockete.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_reset">
   <property name="text">
    <string>Zoom reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+*</string>
   </property>
  </action>
  <action name="actionDisplay_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display grid</string>
   </property>
  </action>
  <action name="actionGrid_scale">
   <property name="text">
    <string>Grid scale</string>
   </property>
  </action>
  <action name="actionSet_from_file">
   <property name="text">
    <string>Set from file...</string>
   </property>
  </action>
  <action name="actionStreched">
   <property name="text">
    <string>Streched</string>
   </property>
  </action>
  <action name="action1024x768">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1024x768</string>
   </property>
  </action>
  <action name="action800x600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>800x600</string>
   </property>
  </action>
  <action name="actionUnselect">
   <property name="text">
    <string>Unselect</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="rockete.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSet_screen_size">
   <property name="text">
    <string>Set screen size...</string>
   </property>
  </action>
  <action name="actionImport_fonts">
   <property name="text">
    <string>Import fonts...</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="rockete.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="rockete.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderingView</class>
   <extends>QWidget</extends>
   <header>RenderingView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rockete.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>menuOpenClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>menuCloseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>menuSaveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeTabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>rocketeClass</receiver>
   <slot>codeTabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_in</sender>
   <signal>triggered()</signal>
   <receiver>renderingView</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_out</sender>
   <signal>triggered()</signal>
   <receiver>renderingView</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_reset</sender>
   <signal>triggered()</signal>
   <receiver>renderingView</receiver>
   <slot>zoomReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUnselect</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>unselectElement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReload</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>menuReloadClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSet_screen_size</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>menuSetScreenSizeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport_fonts</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>menuLoadFonts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>propertyTreeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>rocketeClass</receiver>
   <slot>propertyViewClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>744</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>menuUndoClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>rocketeClass</receiver>
   <slot>menuRedoClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
